### ISSUE TITLE
Improve Display of Wikidata Identifier on Edition Page

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#811]
Currently: https://openlibrary.org/books/OL2388827M/The_tale_of_the_Heike
![image](https://user-images.githubusercontent.com/6251786/36647355-48a32e1c-1a52-11e8-863d-aa3ecd505a47.png)

- Capitalize 'W' in "wikidata"
- Link QID to `https://www.wikidata.org/wiki/{qid}`

**Comment by salman-bhai** [comment]
Hi, 

I'd like to solve this Issue but I have no way of getting data for this particular page. How can I help in resolving this?

**Comment by salman-bhai** [comment]
@cdrini anyways I can contribute to this without having the test data?

**Comment by tfmorris** [comment]
@salman-bhai I think the improvements relate to the template, not the data. 

The two improvements to the template are:

- Change 'wikidata' to 'Wikidata'
- Add an HREF to the Qnnn

**Comment by salman-bhai** [comment]
Yes what I meant to say was that I can make changes in the code but I won't be able to verify if I have done it correctly unless I have a book which has the attribute Wikidata in it?

The test database doesn't have any book which has the ID Number WIkidata!

**Comment by cdrini** [comment]
Hey @salman-bhai , so here's sort of what I was thinking:
- Looking at the html of the example page ( ctrl+u in chrome ) I linked above, you'll see that the Library Thing link looks like `http://www.librarything.com/work/306424`
- That suggests that somewhere in the codebase, there's a string that creates this link. Searching the codebase for "librarything.com" ( which you can also do on GitHub ) gives a single file, which is (probably) where the changes must occur to add formatting for Wikidata.

Hope that helps!

**Comment by tfmorris** [comment]
As we discussed on the call today, these URL templates all live in ./config/edition which was orginally populated from the source tree, but now is an independent thing in the production infogami 

```
$ find . -type f -exec grep -i librarything {} \; -print
            "url": "https://www.librarything.com/work/@@@",
            "name": "librarything",
./openlibrary/plugins/openlibrary/pages/config_edition.page
```

The three at signs (@) are where the identifier gets replaced in the URL template.

The Wikidata template should get added to that page. As I said during the call, I'll do three PRs: 1) migrate current production data to git, 2) add Wikidata (I said I'd add VIAF too, but that makes no sense for config/edition), 3) incorporate the changes from #725  (probably just a revised version of that PR)

**Comment by cdrini** [comment]
This appears still be an issue.

**Comment by hornc** [comment]
@salman-bhai the instructions for importing data are on the wiki at https://github.com/internetarchive/openlibrary/wiki/Getting-Started#importing-test-data

to get this item into the dev environment, the following one-liner can be run from the host:

`docker-compose exec web scripts/copydocs.py /books/OL2388827M --recursive`

I have verified this works in the latest dev environment. It woudl be great if someone could pick this up again.

**Comment by hornc** [comment]
Ok, I have figured this out -- the notes on the closed PR #840 were correct that PR #881 would fix the problem, it just needed to be applied in production, which is why I opened #2062 .  I have just made a production edit to just add the Wikidata entry, which resolves _this_ issue.   #2062 needs to be looked at carefully to get the rest of the changes.
